src/utils/apiCalls.js

```diff
@@ -87,9 +87,9 @@ export const deleteProject = async (projectId) => {
 export const editProject = async (projectId, newName) => {
   const options = {
     method: 'PATCH',
-    body: {
+    body: JSON.stringify({
```

Comment:
Thanks for catching and fixing this! Now I know why I was having issues with some of the tests.